definitions/region/connections.yaml

```
@@ -0,0 +1,192 @@
+# allow directional connections explicitly
+- Austria>Czechia
+- Austria>Germany
+- Austria>Hungary
+- Austria>Italy
+- Austria>Slovakia
+- Austria>Slovenia
+- Austria>Switzerland
+- Belgium>North Sea|Borssele
+- Belgium>France
+- Belgium>Germany
+- Belgium>United Kingdom
+- Belgium>North Sea|Hollandsee Kust
+- Belgium>Luxembourg
+- Belgium>Netherlands
+- North Sea|Borssele>Belgium
+- North Sea|Borssele>North Sea|Helgolander Bucht
+- North Sea|Borssele>North Sea|Hollandsee Kust
+- North Sea|Borssele>North Sea|Nordsoen
+- North Sea|Borssele>North Sea|Sorlige Nordsjo I
+- North Sea|Borssele>North Sea|Sorlige Nordsjo II
+- North Sea|Borssele>North Sea|Utsira Nord
+- Bosnia and Herzegovina>Croatia
+- Bosnia and Herzegovina>Italy
+- Bosnia and Herzegovina>Serbia
+- Bulgaria>Greece
+- Bulgaria>Macedonia
+- Bulgaria>Romania
+- Bulgaria>Serbia
+- Croatia>Hungary
+- Croatia>Italy
+- Croatia>Serbia
+- Croatia>Slovenia
+- Czechia>Germany
+- Czechia>Poland
+- Czechia>Slovakia
+- Denmark>United Kingdom
+- Denmark>Netherlands
+- Denmark>Norway|Sorland
+- Denmark>North Sea|Nordsoen
+- Denmark>Poland
+- Denmark>Sweden
+- North Sea|Dogger Bank>North Sea|Borssele
```

The North Sea regions are defined for EMPIRE v2, please fix everywhere.
```suggestion
- EMPIRE v2|North Sea|Dogger Bank>EMPIRE v2|North Sea|Borssele
```

---

Should be done now, changed it and pushed it in my fork